### PR TITLE
At order a text is sent to the user

### LINKED ISSUE #55: Estimated Delivery Time - twilio
Possible integration with google maps
estimate from address to address

### PR DESCRIPTION
**Pull request by applegrain**
closes #55 

- when an order is placed, the user gets a text message saying the order is on the way including the delivery time.

### CODE REVIEW

File: app/controllers/orders_controller.rb [hunk 1]
```diff
@@ -47,7 +47,11 @@ def send_text_message
     @twilio_client.account.messages.create(
       from: "+1#{twilio_phone_number}",
       to: "+1#{send_to}",
-      body: "Your order is on it's way! - Dinner's Ready"
+      body: "Your order is on it's way! Dinner will arrive in #{estimated_delivery_time} minutes. - Dinner's Ready"
```
Comment (mrjaimisra) [inline app/controllers/orders_controller.rb]: good idea interpolating the time

File: spec/models/user_spec.rb [hunk 2]
```diff
@@ -49,8 +49,13 @@
...
 
-    it "can get deliver if they live within 50 miles of the store" do
+    it "can get delivery if they live within 50 miles of the store" do
       expect(user.valid_delivery?).to eq true
     end
+
+    it "can get estimated delivery time" do
+      allow_any_instance_of(User).to receive(:delivery_time).and_return(7)
+      expect(user.delivery_time).to eq 7
+    end
```
Comment (mrjaimisra) [inline spec/models/user_spec.rb]: it looks like you did stub the time the get it to pass

**Comment by applegrain** [comment]
I'm having troubles figuring out how to properly test the #valid_delivery and #delivery_time. I have checked it in the browser to make sure it works they way we want it to, and at this point I'd say it's ok to merge without tests.

File: spec/models/user_spec.rb (hunk 2 above)
Comment (noobjey) [inline spec/models/user_spec.rb]: After what josh said about testing 3rd party api's i would say its ok until we learn how to do it in module 3

**Pull request merged**